<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgispindeleinstellung</class>
 <widget class="QDialog" name="dlgispindeleinstellung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>454</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Einstellungen iSpindel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Datenbankverbindung</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Anmeldedaten Datenbank</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_DbUsername">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Passwort</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_DbPassword">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Datenbank</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Name der Datenbank</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_NameDatabase">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP des Host</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_IpDatabase">
            <property name="inputMask">
             <string>000.000.000.000</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>(ODBC) Treiber</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_OdbcTreiber">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnTestConnection">
           <property name="text">
            <string>Teste die Verbindung</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveClose">
           <property name="text">
            <string>Speichern und schließen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Einstellungen Datenimport</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_TabelleKalibrierung">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Wähle Tabelle mit den Kalibrierdaten:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Wähle Tabelle mit den Messdaten:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_TabelleMessdaten"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Kalibrierung</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Wähle die Spindel die beschrieben werden soll</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Spindel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxChooseSpindel"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Parameter</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Setze die Parameter zu der ausgewählten Spindel</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEditX_2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>* x²</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditX_1"/>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string> * x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditX_0"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butSaveParameter">
         <property name="text">
          <string>Speichern in Datanbank</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Reset Flag</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Setze jetzt ein Reset Flag für die oben ausgewählte Spindel.
Damit können nachfolgende Messpunkte einfacher separiert 
werden. Diese Aktion empfielt sich immer nach
dem Anstellen eines neuen Sudes.
Die Sudnummer dient lediglich der besseren Visualiserung.
Sind mehrere Spindel verfügbar, muss jede Spindel einzeln
ausgewählt und gesetzt werden!</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Sudnummer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxReceipeNr">
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="butSetResetflag">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Setze Reset Flag</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
